aries-site/src/pages/components/card.mdx

```diff
@@ -51,9 +57,14 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- When presenting high-level, summarized pieces of information.
+- When presenting an item or collection of items that can best be consumed visually.
+
 ## When not to use
```

Comment:
I'm wondering if we just make these as visual "don'ts". Considering this page is already text heavy, I like the idea of being able to quickly and visually look at examples of when i'm not supposed to use a card.

Reply:
Agree, I think we can pull in some of the visual examples that you worked on. Will give that a go.